### PR TITLE
f-services@0.7.0-remove-method

### PR DESCRIPTION
**Pull request by smilburn1990**
### Removed 
- `transformLocale` method as it was not working

### CODE REVIEW

**Review by dandel10n — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

File: packages/f-services/src/services/shared.service.js [hunk 1]
```diff
@@ -13,18 +13,15 @@
  * @returns {String}
  */
 const getLocale = (tenantConfigs, tenantString, globalTenant) => {
-    const locale = tenantString === '' ? globalTenant?.locale : tenantString;
-
-    // makes sure the locale is the same format as the tenants - 'en-GB'
-    let transformedLocale = `${locale.slice(0, -3)}${locale.slice(locale.lastIndexOf('-')).toUpperCase()}`;
+    let locale = tenantString === '' ? globalTenant?.locale : tenantString;
```
Comment (kevinrodrigues) [inline packages/f-services/src/services/shared.service.js:16]: Do the tests need updating?

File: packages/f-services/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+0.7.0
+------------------------------
+*September 26, 2019*
```
Comment (kevinrodrigues) [inline packages/f-services/CHANGELOG.md]: Date needs bumping. 27th 📅

File: packages/f-services/src/services/shared.service.js (hunk 1 above)
Comment (smilburn1990) [reply packages/f-services/src/services/shared.service.js:16]: Nope - Didn't get a chance to update them when I added it in the first place 👍

**Review by kevinrodrigues — APPROVED** [review]

File: packages/f-services/CHANGELOG.md (hunk 2 above)
Comment (dandel10n) [reply packages/f-services/CHANGELOG.md]: Technically changes were made yesterday and PR can wait days to be approved so I vote for not bumping the date if it is the latest change and there are no conflicts with other PRs

**Review by dandel10n — APPROVED** [review]

File: packages/f-services/CHANGELOG.md (hunk 2 above)
Comment (kevinrodrigues) [reply packages/f-services/CHANGELOG.md]: I agree but this is what we have done for all fozzie components in the past.

**Pull request merged**